overlays/python36/Dockerfile

```diff
@@ -1,26 +1,35 @@
 FROM quay.io/thoth-station/s2i-thoth-ubi8-py36:v0.32.3
-LABEL name="s2i-minimal-notebook:latest" \
-      summary="Minimal Jupyter Notebook Source-to-Image for Python 3.6 applications." \
-      description="Notebook image based on Source-to-Image.These images can be used in OpenDatahub JupterHub." \
-      io.k8s.description="Notebook image based on Source-to-Image.These images can be used in OpenDatahub JupterHub." \
-      io.k8s.display-name="Minimal Notebook Python 3.6-ubi8 S2I" \
-      io.openshift.expose-services="8888:http" \
-      io.openshift.tags="python,python36" \
-      authoritative-source-url="https://quay.io/thoth-station/s2i-minimal-notebook" \
-      io.openshift.s2i.build.commit.ref="master" \
-      io.openshift.s2i.build.source-location="https://github/thoth-station/s2i-minimal-notebook" \
-      io.openshift.s2i.build.image="quay.io/thoth-station/s2i-thoth-ubi8-py36:v0.32.0"
 
-ENV JUPYTER_ENABLE_LAB="1" \
+ARG IMAGE_NAME="quay.io/thoth-station/s2i-minimal-notebook"
+ARG IMAGE_TAG="latest"
```

Comment:
Why are the UBI images done this way with an arguement instead of passing the variable through?

Reply:
Normally args are used for passing value during buildtime.
 removed it as we dont need it.